src/h5web/visualizations/RawVis.tsx

```diff
@@ -11,12 +11,12 @@ function RawVis(props: Props): ReactElement {
   const valueAsStr = JSON.stringify(value, null, 2);
 
   if (valueAsStr.length > 1000) {
-    // eslint-disable-next-line no-console
-    console.log(valueAsStr);
+    console.log(value); // eslint-disable-line no-console
```

Comment:
Logging the raw value instead of the stringified value.

Reply:
Good call :+1: